### PR TITLE
Add contributing documents

### LINKED ISSUE #32: create contributor guide (Due: 2024-03-01)
In preparation for GSOC and otherwise growing the body of contributors working on this project, please get together and define agreed standards of how to be a good contributor to the project. 

Things to consider
- Code style guidelines
- Best practices on reporting bugs and opening new issues
- Other relevant standards that would make shared work go more smoothly

(Including @aaron-wu1 because the same contributor guide would apply for the software/dirtviz side)

### PR DESCRIPTION
**Pull request by jmadden173**
Added `CONTRIBUTING.md` and added supporting documents. Also added templates for creating issues/PRs based on documents.

Closes #32

### CODE REVIEW

File: README.md
Comment (stevegtaylor) [inline README.md:1]: Code is licensed under MIT should be code is licensed under UCSC

**Review by stevegtaylor — COMMENTED** [review]
-Are we going to ask contributors for a brief blurb about who they are?
-added comment about incorrect license 

everything else looks goo

File: README.md
Comment (aaron-wu1) [reply README.md:1]: is there somewhere where I can copy the UCSC license for dirtviz or is that okay under the MIT license?

**Comment by stevegtaylor** [comment]
license document referenced in the readme does not exist

**Pull request merged**